resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders API
+  contact: {}
+  version: '1.0'
+  description: |
+    # Overview
+    Use the CTC Traders API to send departure and arrival movement to the New Computerised Transit System (NCTS). 
+    Retrieve messages sent from customs offices of departure and destination.

```

Comment:
Two issues:

- The second sentence looks disjointed from the first sentence - use either a bullet list or merge the two sentences.
- GDS style guidelines advise against using "retrieve" instead of "get".

Suggestion:
  Use the CTC Traders API to send departure and arrival movement to the New Computerised Transit System (NCTS) and to get messages sent from customs offices of departure and destination.